src/nls/root/strings.js

```diff
@@ -585,14 +585,14 @@ define({
     "CMD_RESTART_NODE"                          : "Restart Node",
     "CMD_SHOW_ERRORS_IN_STATUS_BAR"             : "Show Errors in Status Bar",
     "CMD_OPEN_BRACKETS_SOURCE"                  : "Open {APP_NAME} Source",
-    "ERROR_CREATING_LAUNCH_SCRIPT"              : "An error occured while creating {APP_NAME} command line tool at <code>/usr/local/bin</code>. Please refer to <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments#troubleshooting'>command line</a> wiki for troubleshooting.<br/><br/>Reason: ",
+    "ERROR_CREATING_LAUNCH_SCRIPT"              : "An error occured while creating {APP_NAME} command line tool at <code>/usr/local/bin</code>. Please refer to our <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments#troubleshooting'>command line</a> wiki for troubleshooting.<br/><br/>Reason: {0}",
     "ERROR_CLTOOLS_RMFAILED"                    : "Failed to remove existing {APP_NAME} symlink at <code>/usr/local/bin.</code>",
     "ERROR_CLTOOLS_MKDIRFAILED"                 : "Failed to create <code>/usr/local/bin</code> directory structure.",
     "ERROR_CLTOOLS_LNFAILED"                    : "Failed to create {APP_NAME} symlink at <code>/usr/local/bin</code>.",
     "ERROR_CLTOOLS_SERVFAILED"                  : "Failed to create authorization object.",
     "ERROR_CLTOOLS_NOTSUPPORTED"                : "{APP_NAME} command line tool installation is not supported on this OS.",
 
-    "LAUNCH_SCRIPT_CREATE_SUCCESS"              : "Command line tool successfully installed! Now you can easily launch {APP_NAME} from command line using <code>{APP_NAME} myFile.txt</code> or <code>{APP_NAME} myFolder</code>. <br/><br/>Please refer to <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments'>command line</a> wiki for more infomation.",
+    "LAUNCH_SCRIPT_CREATE_SUCCESS"              : "Command line tool successfully installed! Now you can easily launch {APP_NAME} from command line using <code>brackets myFile.txt</code> or <code>brackets myFolder</code>. <br/><br/>Please refer to our <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments'>command line</a> wiki for more infomation.",
```

Comment:
@nethip I think we should change it to actually create a lowercase symlink file too (change the brackets-shell code).  For several reasons:

* In case-sensitive filesystems (which can optionally be enabled on Mac), you won't be able to type `brackets` otherwise.
* Even on a default Mac filesystem, you can't type `brac`-Tab to autocomplete the command line (`Brac`-Tab works, since it matches what's on disk -- but people won't know that since our help string now pretends it's lowercase).
* If we do this same feature for Linux later, it'll have to be lowercase because Linux is case-sensitive by default.
* Everything else in `/usr/local/bin` is lowercase, so it makes us look bad to be the only one ignoring the convention.